### PR TITLE
feat: Add extraEnvs to operator helm chart

### PR DESCRIPTION
**Pull request by desaintmartin**
Fixes https://github.com/zalando/postgres-operator/issues/2672

Inspired from same feature from UI chart.

### CODE REVIEW

**Comment by FxKu** [comment]
Can you remove the comments about UI?

File: charts/postgres-operator/values.yaml [hunk 1]
```diff
@@ -504,6 +504,25 @@ readinessProbe:
   initialDelaySeconds: 5
   periodSeconds: 10
 
+# configure extra ENVs
+# Extra ENVs are writen in kubernetes format and added "as is" to the pod's env variables
+# https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/
+# https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#environment-variables
+# UI specific env variables can be found here: https://github.com/zalando/postgres-operator/blob/master/ui/operator_ui/main.py
```
Comment (FxKu) [inline charts/postgres-operator/values.yaml:511]:
```suggestion
```

File: charts/postgres-operator/values.yaml [hunk 2]
```diff
@@ -504,6 +504,25 @@ readinessProbe:
...
+extraEnvs:
+  []
+  # Exemple of settings to make snapshot view working in the ui when using AWS
```
Comment (FxKu) [inline charts/postgres-operator/values.yaml:514]: this comment doesn't match the examples

Comment (desaintmartin) [reply charts/postgres-operator/values.yaml]: you're right! Here it is, fixed.

**Comment by FxKu** [comment]
:+1:

**Comment by idanovinda** [comment]
👍

**Comment by pat-trunk-io** [comment]
Shouldn't the indent be 10 not 12 to match the indentation of resources below?
```{{- .Values.extraEnvs | toYaml | nindent 12 }}```

**Pull request merged**